config/db.php

```diff
@@ -0,0 +1,6 @@
+<?php
```

Comment:
Is this file or the init.php file still necessary with the removal of the extra providers?